assets/src/dashboard/app/views/exploreTemplates/header/test/header.js

```diff
@@ -158,11 +161,35 @@ describe('Explore Templates <Header />', function () {
             pageSize: { width: 200, height: 300 },
           }}
         />
-      </LayoutProvider>
+      </LayoutProvider>,
+      { enableInProgressTemplateActions: true }
     );
     fireEvent.click(getAllByText('Popular')[0].parentElement);
     fireEvent.click(getByText('Recent'));
 
     expect(setSortFn).toHaveBeenCalledWith('recent');
   });
+
+  it('should not render with search when enableInProgressTemplateActions is false', function () {
+    const { queryAllByRole } = renderWithThemeAndFlagsProvider(
+      <LayoutProvider>
+        <Header
+          filter={{ value: TEMPLATES_GALLERY_STATUS.ALL }}
+          totalTemplates={8}
+          search={{ keyword: '', setKeyword: jest.fn }}
+          templates={fakeTemplates}
+          sort={{
+            value: TEMPLATES_GALLERY_SORT_OPTIONS.POPULAR,
+            set: jest.fn,
+          }}
+          view={{
+            style: VIEW_STYLE.GRID,
+            pageSize: { width: 200, height: 300 },
+          }}
+        />
+      </LayoutProvider>,
+      { enableInProgressTemplateActions: false }
+    );
+    expect(queryAllByRole('input')).toHaveLength(0);
```

Comment:
This test creates a false positive as written. The role for inputs are `textbox`.

When this is `expect(queryAllByRole('textbox')).toHaveLength(0);`

Then the test correctly works depending on if the flag is false, then length 0, true, then length is 1.

Reply:
thanks for fixing this